src/test/examples.ml

```diff
@@ -154,9 +164,7 @@ let downloader () =
         if_seq (filename_ov =$= no_value)
           ~t:begin
             let filename =
-              url >> exec ["sed"; "s:.*/\\([^\\?\\/]*\\)\\?.*:\\1:"]
-              |> output_as_string
-            in
+              no_newline_sed ~input:url "s/.*\\/\\([^?\\/]*\\).*/\\1/" in
```

Comment:
I didn't manage to get to something portable with the `s: ... : .. :` syntax.
Also I don't "see" it there: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html